src/sif/pedestriancost.cc

```diff
@@ -50,7 +50,7 @@ constexpr uint32_t kDefaultMaxGradeWheelchair = 12; // Conservative for now...
 constexpr uint8_t kDefaultMaxHikingDifficulty = 1; // T1 (kHiking)
 constexpr float kModeFactor = 1.5f;                // Favor this mode?
 constexpr float kDefaultWalkwayFactor = 0.9f;      // Slightly favor walkways
-constexpr float kDefaultSideWalkFactor = 0.95f;    // Slightly favor sidewalks
+constexpr float kDefaultSideWalkFactor = 0.25f;    // Slightly favor sidewalks
```

Comment:
my experience is that factors this low can cause issues. Does this not work at a value of 0.9 or 0.8?